PolyEngine/Core/Core.vcxproj

```diff
@@ -130,9 +130,7 @@
     </Link>
   </ItemDefinitionGroup>
   <ItemGroup>
-    <ClCompile Include="Src\BaseObject.cpp" />
     <ClCompile Include="Src\Color.cpp" />
-    <ClCompile Include="Src\CorePCH.cpp">
```

Comment:
This is weird, precompiled headers should not need .cpp files... Could you give me some resources to read on MSVC support for precompiled headers?